plugins/services/src/js/events/ServiceDeleteActions.js

```diff
@@ -0,0 +1,23 @@
+import CosmosPackagesActions from "#SRC/js/events/CosmosPackagesActions";
+import MarathonActions from "./MarathonActions";
+import Framework from "../structs/Framework";
+
+const ServiceDeleteActions = {
```

Comment:
Why do we add service **delete** actions? Let's please stick with the pattern and add general service actions.

Reply:
@orlandohohmeier renamed to serviceActions